go.mod

```diff
@@ -4,7 +4,7 @@ go 1.17
 
 require (
 	github.com/HdrHistogram/hdrhistogram-go v1.1.2
-	github.com/gocql/gocql v1.2.1
+	github.com/gocql/gocql v1.5.2
```

Comment:
Shouldn't the line below:

```
replace github.com/gocql/gocql => github.com/scylladb/gocql v1.7.3
```

get changed instead of this line? I'm not sure this change actually achieves anything?

Reply:
my bad, fixed that